presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestPrestoContainerRemoteFunction.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.nativeworker;
+
+import com.facebook.presto.tests.AbstractTestQueryFramework;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class TestPrestoContainerRemoteFunction
+        extends AbstractTestQueryFramework
+{
+    @Override
+    protected ContainerQueryRunnerWithFunctionServer createQueryRunner()
+            throws Exception
+    {
+        return new ContainerQueryRunnerWithFunctionServer();
+    }
+
+    @Test
+    public void testRemoteFunctions()
+    {
+        assertEquals(
+                computeActual("select remote.default.abs(-10)")
+                        .getMaterializedRows().get(0).getField(0).toString(),
+                "10");
+        assertEquals(
+                computeActual("select remote.default.abs(-1230)")
+                        .getMaterializedRows().get(0).getField(0).toString(),
+                "1230");
+        assertEquals(
+                computeActual("select remote.default.second(CAST('2001-01-02 03:04:05' as timestamp))")
+                        .getMaterializedRows().get(0).getField(0).toString(),
+                "5");
+        assertEquals(
+                computeActual("select remote.default.length(CAST('AB' AS VARBINARY))")
+                        .getMaterializedRows().get(0).getField(0).toString(),
+                "2");
+        assertEquals(
```

Comment:
Please add test for scalar functions which is available only in remote function server, not in native. like from_base32
```
presto> select remote.default.from_base32(CAST('MFRGG===' AS VARBINARY));
presto> select remote.default.to_base32(CAST('abc' AS VARBINARY));
```